input/data/2012/2012A%26A...537A.114A/tev-000090.yaml

```diff
@@ -1,4 +1,24 @@
 source_id: 90
 reference_id: 2012A&A...537A.114A
 
-# TODO: add data
+data:
+  livetime: 33.8
+  significance: 20
+
+pos:
+  ra: {val: 16h46m50s, err: 27s, }
+  dec: {val: -45d49m12s, err: 7s, }
+
+morph:
+  type: gauss
```

Comment:
This is a very curious case. No fitted extension was published in the paper. It's not that we missed this, I noticed this years ago and have an email thread with the corresponding author.

Here's my suggestion what to put in the file for now as a comment to explain:
```
# This paper only gives the position, but not the extension or flux of the morphology fit.
# Corresponding author Stefan Ohm confirmed this and pointed out the full results
# in HESS-internal pages.
# We're not adding this information here, because it's not public information.
```